msw/build-nsi.sh

```diff
@@ -190,11 +190,11 @@ case $nsis_exit in
 esac
 
 # copy installer custom artwork
-cp "${SCRIPTDIR}/installer-art/big.bmp" ${OUTDIR}/big.bmp
-cp "${SCRIPTDIR}/installer-art/small.bmp" ${OUTDIR}/small.bmp
-cp "${SCRIPTDIR}/installer-art/small-un.bmp" ${OUTDIR}/small-un.bmp
-cp "${SCRIPTDIR}/installer-art/pd-un.ico" ${OUTDIR}/pd-un.ico
-cp "${SCRIPTDIR}/../tcl/pd.ico" ${OUTDIR}/pd.ico
+cp ${SCRIPTDIR}/installer-art/big.bmp ${OUTDIR}/big.bmp
+cp ${SCRIPTDIR}/installer-art/small.bmp ${OUTDIR}/small.bmp
+cp ${SCRIPTDIR}/installer-art/small-un.bmp ${OUTDIR}/small-un.bmp
+cp ${SCRIPTDIR}/installer-art/pd-un.ico ${OUTDIR}/pd-un.ico
+cp ${SCRIPTDIR}/../tcl/pd.ico ${OUTDIR}/pd.ico
```

Comment:
that doesn't look right.
why are the quotes removed? (was that me??)
there should *also* be quotes around the `${OUTDIR}/...`

Reply:
i did a `force push` that didn't change anything (my bad) and 1minute after you `force push`ed the fix. while I was trying to fix my bad I got lost in history. I accidentally tried to fix conflicts but something went wrong.

Reply:
now fixed via 687c970